Resources/Textures/Clothing/OuterClothing/Hardsuits/brigmedic.rsi/meta.json

```
@@ -2,6 +2,7 @@
   "version": 1,
   "license": "CC-BY-SA-3.0",
   "copyright": "Made by kuro(388673708753027083)",
```

Biome (1.9.4)

[error] The key copyright was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)